clusters/build-clusters/01_cluster/ci-search/deploy.yaml

```diff
@@ -104,6 +118,7 @@ spec:
     spec:
       accessModes:
       - ReadWriteOnce
+      storageClassName: gp3-csi
```

Comment:
removed the line because of ?
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/31964/pull-ci-openshift-release-master-build01-dry/1565779951695695872#1:build-log.txt%3A43

I guess you can override the presubmit and oc-replace with `gp3-csi` locally

Reply:
Yeah, apparently that is immutable with statefulset sets, so I had to punt on changing it. If we change the default storage class on our systems to gp3, it should take care of the scenario where this PVCs are deleted.
For the time being, I've converted them from gp2 to gp3 directly in AWS -- this is a transparent operation for the instance / workload.